testing/librtlsdr/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor: Jared Szechy <jared.szechy@gmail.com>
+# Maintainer: Jared Szechy <jared.szechy@gmail.com>
+pkgname=librtlsdr
+pkgver=0.5.3
+pkgrel=0
+pkgdesc="Software defined radio receiver for Realtek RTL2832U"
+url="https://osmocom.org/projects/sdr/wiki/rtl-sdr"
+arch="all"
+license="GPL2"
+depends="libusb"
+depends_dev="libusb-dev"
+makedepends="$depends_dev cmake"
+install=""
+subpackages="$pkgname-dev rtl-sdr:tools"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/steve-m/$pkgname/archive/v$pkgver.tar.gz"
+
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	mkdir -p build
+	cd build
+	cmake -DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX:PATH=/usr \
+		-DDETACH_KERNEL_DRIVER=ON \
+		-Wno-dev ..
+	make
+}
+
+package() {
+	cd "$builddir"/build
+
+	make DESTDIR="$pkgdir" install
+}
+
+tools() {
+	pkgdesc="Software defined radio receiver for Realtek RTL2832U (tools)"
+	mkdir -p "$subpkgdir"/usr
+	mv "$pkgdir"/usr/bin "$subpkgdir"/usr/
+}
+
+md5sums="9f6d8c4b5e1998305d0346689b43db98  librtlsdr-0.5.3.tar.gz"
```

Comment:
md5sums and sha256sums are no longer used, you could use recent `abuild checksum` to update the checksums and see these go away.